### PR TITLE
Updating citing Coq in FAQ.

### PR DESCRIPTION
**Pull request by herbelin**
This is [BZ5766](https://coq.inria.fr/bugs/show_bug.cgi?id=5766) for its doc part. (But no change of Readme.md)

### CODE REVIEW

**Review by gares — APPROVED** [review]

**Comment by ejgallego** [comment]
Regarding README.md , it is common practice in "research" github projects to indicate that information in the landing page; I suggest we add a quick note indeed.

**Comment by herbelin** [comment]
@ejgallego: regarding the README.md, could you make a proposal of what you have in mind?

**Comment by ejgallego** [comment]
@herbelin I'd take some inspiration from what Clément does here: https://github.com/cpitclaudel/company-coq regading zenodo, bibtex, etc... What do you think?

**Comment by herbelin** [comment]
@ejgallego: you mean this [item](https://github.com/cpitclaudel/company-coq#citing). No objection (on my side).

Afaiu, `company-coq-cite` in coq-company has to be aware of the Coq bibtex. Do we have something to do here or @cpitclaudel (Hi!) is taking care of dynamically extracting from somewhere a relevant bibtex?

**Comment by ejgallego** [comment]
> @ejgallego: you mean this item. No objection (on my side).

Cool, what do you think about having a Zenodo entry for the manual?

> Afaiu, company-coq-cite in coq-company has to be aware of the Coq bibtex. Do we have something to do here or @cpitclaudel (Hi!) is taking care of dynamically extracting from somewhere a relevant bibtex?

Oh interesting! Maybe we should add a canonical "bib" file to the repository or website so people could depend on it? [Maybe getting a zenodo entry serves this purpose, so you can do \cite{zenodo:XXXX}]

**Comment by herbelin** [comment]
No opinion on Zenodo which I'm just discovering. What is this?

**Comment by ejgallego** [comment]
@herbelin it is a "paper" repository, aimed to improve access to scientific documents; I am not expert either, and there are other alternatives [I just picked Zenodo as Clément uses it]

The main advantage is that it would provide a DOI for the latest Coq Reference manual.

**Comment by Zimmi48** [comment]
I'd suggest you have a look at the Coqdev thread "Citing Coq" from May 11th, 2015, started by @cpitclaudel himself.

**Comment by ejgallego** [comment]
Cool @Zimmi48 , here's the link to the archive: https://sympa.inria.fr/sympa/arc/coqdev/2015-05/msg00025.html

**Comment by herbelin** [comment]
All the directions mentioned there look worth to me. If it is a fatigue duty for each of us to register the manual on HAL, and to get a DOI at Zenodo, shall we draw straws? (And add an item on the list of release todos).

**Comment by cpitclaudel** [comment]
Hi @herbelin! It's a static string, hardcoded into company-coq; happy to update it!

**Comment by Zimmi48** [comment]
> If it is a fatigue duty for each of us to register the manual on HAL, and to get a DOI at Zenodo, shall we draw straws? (And add an item on the list of release todos).

In fact, Zenodo looks pretty awesome. There is a GitHub feature which allows to automatically generate a new DOI at each release. I've activated it to test it. Normally we should thus get a DOI when we release Coq 8.7+beta2 later today. Once we have a DOI, we also get a badge that we can add to the README (like on Company-Coq's README). If instead of updating the FAQ after each release, we say something along the lines of "To get the latest information on how to cite Coq, please refer to the following page..." we won't even have to add a new item in the release to-do list.

**Comment by herbelin** [comment]
> If instead of updating the FAQ after each release, we say something along the lines of "To get the latest information on how to cite Coq, please refer to the following page..." we won't even have to add a new item in the release to-do list.

Indeed, that would certainly make maintenance easier.

> It's a static string, hardcoded into company-coq; happy to update it!

And maybe, a file containing the bibtex entry could be put somethere in the `distrib` directory of each version, and in the install directory of each coq package so that a tool like coq-company can automatically get the  bibtex entry corresponding to the Coq version the user is using.

**Comment by aspiwack** [comment]
> There is a GitHub feature which allows to automatically generate a new DOI at each release. I've activated it to test it.

Just an side advice: do take note somewhere in the coq repository of _everything_  that you've configured in Github. It will help whoever's in charge at the time where the config is lost (crash on github, hack, migration) not to go completely insane (hey, who knows, it may be you, indefinite reader!).

File: doc/faq/FAQ.tex [hunk 1]
```diff
@@ -2413,15 +2413,14 @@ \section{Publishing tools}
...
-@Manual{Coq:manual,
-  title =        {The Coq proof assistant reference manual},
-  author =       {\mbox{The Coq development team}},
-  organization = {LogiCal Project},
-  note =         {Version 8.2},
-  year =         {2009},
-  url =          "http://coq.inria.fr"
+@manual{Coq:manual,
+  author      = {{Coq} {Development} {Team}, The},
+  title       = {The {Coq} Proof Assistant Reference Manual, version 8.6},
```
Comment (Zimmi48) [inline doc/faq/FAQ.tex]: Can you put 8.7.0 instead?

File: doc/faq/FAQ.tex [hunk 2]
```diff
@@ -2413,15 +2413,14 @@ \section{Publishing tools}
...
+  month       = Dec,
+  year        = {2016},
```
Comment (Zimmi48) [inline doc/faq/FAQ.tex]: Oct 2017.

**Review by Zimmi48 — CHANGES_REQUESTED** [review]
Given how we are just about to release 8.7.0 let's make sure this is not almost immediately outdated.

I have another question: why do you have to update both this file and open coq/www#47? What is the purpose of the present file?

**Comment by herbelin** [comment]
> I have another question: why do you have to update both this file and open coq/www#47? What is the purpose of the present file?

The present file is/was the original source of the FAQ. It is still used to produce the FAQ.pdf for instance.

At some time, Denis Cousineau made a nice-looking html version of the FAQ, but he made it by hand, so that the synchronization between the "master" tex copy and the copy on the web site is not any more mechanized.

> Given how we are just about to release 8.7.0 let's make sure this is not almost immediately outdated.

Done, but knowing that this is still not a very robust way of proceeding.

> Just an side advice: do take note somewhere in the coq repository of everything that you've configured in Github. It will help whoever's in charge at the time where the config is lost (crash on github, hack, migration) not to go completely insane (hey, who knows, it may be you, indefinite reader!)

Anyone github expert willing to do that?

**Comment by Zimmi48** [comment]
On the Zenodo topic, I'll open an issue to discuss it. I don't think it would be wise to do a last minute change on the way we recommend citing Coq just before the 8.7.0 release.

I pretty worried about the non-synchronization between the HTML and TeX FAQ... (that's not going to help keeping it up-to-date if changes have to be duplicated). What about just dropping the TeX version? I suppose no one compiles it to PDF anymore...

**Comment by herbelin** [comment]
> What about just dropping the TeX version? I suppose no one compiles it to PDF anymore...

Yes, that's probably ok. I don't know the general opinion though.

(Note the dependency in an xfig figure though, as master for an eps file - doc/FAQ/axioms.fig. I was pretty proud of this synthesis of the dependency graph of axioms that can be added to Coq. It should be updated, extended, and put on the web site too.)

**Review by Zimmi48 — APPROVED** [review]

**Comment by Zimmi48** [comment]
OK let's merge this as a temporary fix for 8.7, but then discuss the future of the FAQ, and in particular its removal from the sources, trying to save as much as can be (in particular this figure that you mentioned).

**Pull request merged**